README.md

```diff
@@ -1,13 +1,6 @@
-# Medplum Hello World
+# Foo Clinical
 
-This example app demonstrates the following:
-
-- Creating a new React app with Vite and TypeScript
-- Adding Medplum dependencies
-- Adding basic URL routing
-- Using the Medplum client to search for FHIR resources
-- Using Medplum GraphQL queries
-- Using Medplum React controls to display FHIR data
+This is a sample EHR-like application using Medplum. Medplum is a developer platform for medical apps.
```

Comment:
~~This~~ -> Foo Clinical
- Add the current URL to Foo Clinical.

I would add that this project is to show how simple it is to build your own EHR application using Medplum. Your customer is someone considering building using Medplum so you want to work backwards from why they are looking at your project. Sell them!

Say what parts of Medplum you used to build this, i.e. Medplum App and UI Components.

> Medplum App - This is a web application where can you can view your data, perform basic editing tasks. You can also use the Medplum App to manage basic workflows.
> UI Component Library - Medplum ships with a system of React components designed to help you quickly develop custom healthcare applications

I took that from their main README.